test/e2e/tests/encrypt-decrypt.spec.js

```diff
@@ -99,4 +94,91 @@ describe('Encrypt Decrypt', function () {
       },
     );
   });
+
+  it('should show balance correctly as ETH', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+        await driver.openNewPage('http://127.0.0.1:8080');
+
+        // ------ Get Encryption key and display ETH ------
+        await driver.clickElement('#getEncryptionKeyButton');
+        await driver.waitUntilXWindowHandles(3);
+        const windowHandles = await driver.getAllWindowHandles();
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.waitForSelector({
+          css: '.request-encryption-public-key__header__text',
+          text: 'Request encryption public key',
+        });
+        // Account balance is converted properly
+        const accountBalanceLabel = await driver.findElement(
+          '.request-encryption-public-key__balance-value',
+        );
+        assert.equal(await accountBalanceLabel.getText(), '25 ETH');
+      },
+    );
+  });
+
+  it('should show balance correctly as Fiat', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        await driver.clickElement('.account-menu__icon');
+        // change the settings now
+        await driver.clickElement({ text: 'Settings', tag: 'div' });
+        // goes to general page
+        await driver.clickElement({ text: 'General', tag: 'div' });
+        // Switch to fiat currency
+        await driver.clickElement('#fiat-primary-currency');
```

Comment:
You can enable this setting in the test fixures

Reply:
It would be something along the lines of this :
```
          .withPreferencesController({
            preferences: {
              showFiatInTestnets: true,
              useNativeCurrencyAsPrimaryCurrency: false,
            },
          })
```
* This hasn't been verified for accuracy

Reply:
see: [Generating fixture data](https://github.com/MetaMask/metamask-extension/blob/develop/docs/generating-fixture-data.md)

Reply:
Thanks Peter, that's super useful. i was desperate to click buttons one by one to change the settings :P